// _@vuepress_core@1.7.1@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/licop/Documents/GitHub/What_is_FE/node_modules/_@vuepress_core@1.7.1@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-54918cf7",
    path: "/development-process/Git/git%E5%91%BD%E4%BB%A4.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-54918cf7").then(next)
    },
  },
  {
    path: "/development-process/Git/git命令.html",
    redirect: "/development-process/Git/git%E5%91%BD%E4%BB%A4.html"
  },
  {
    path: "/development-process/Git/git命令.html",
    redirect: "/development-process/Git/git%E5%91%BD%E4%BB%A4.html"
  },
  {
    name: "v-3d79a3a3",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3d79a3a3").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-bbf823d8",
    path: "/development-process/CI-CD/CI-CD%E4%BB%A5%E5%8F%8Atravis%E7%9A%84%E4%BD%BF%E7%94%A8.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-bbf823d8").then(next)
    },
  },
  {
    path: "/development-process/CI-CD/CI-CD以及travis的使用.html",
    redirect: "/development-process/CI-CD/CI-CD%E4%BB%A5%E5%8F%8Atravis%E7%9A%84%E4%BD%BF%E7%94%A8.html"
  },
  {
    path: "/development-process/CI-CD/CI-CD以及travis的使用.html",
    redirect: "/development-process/CI-CD/CI-CD%E4%BB%A5%E5%8F%8Atravis%E7%9A%84%E4%BD%BF%E7%94%A8.html"
  },
  {
    name: "v-30d5bc5c",
    path: "/development-process/Linux/linux%20%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-30d5bc5c").then(next)
    },
  },
  {
    path: "/development-process/Linux/linux 常用命令.html",
    redirect: "/development-process/Linux/linux%20%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4.html"
  },
  {
    path: "/development-process/Linux/linux 常用命令.html",
    redirect: "/development-process/Linux/linux%20%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4.html"
  },
  {
    name: "v-9aceae9a",
    path: "/development-process/Git/%E4%BD%BF%E7%94%A8omyzsh%20git%E6%8F%92%E4%BB%B6%E7%9A%84%E5%88%AB%E5%90%8D.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-9aceae9a").then(next)
    },
  },
  {
    path: "/development-process/Git/使用omyzsh git插件的别名.html",
    redirect: "/development-process/Git/%E4%BD%BF%E7%94%A8omyzsh%20git%E6%8F%92%E4%BB%B6%E7%9A%84%E5%88%AB%E5%90%8D.html"
  },
  {
    path: "/development-process/Git/使用omyzsh git插件的别名.html",
    redirect: "/development-process/Git/%E4%BD%BF%E7%94%A8omyzsh%20git%E6%8F%92%E4%BB%B6%E7%9A%84%E5%88%AB%E5%90%8D.html"
  },
  {
    name: "v-3f7a5014",
    path: "/development-process/NPM/%E5%85%B3%E4%BA%8Enpx.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3f7a5014").then(next)
    },
  },
  {
    path: "/development-process/NPM/关于npx.html",
    redirect: "/development-process/NPM/%E5%85%B3%E4%BA%8Enpx.html"
  },
  {
    path: "/development-process/NPM/关于npx.html",
    redirect: "/development-process/NPM/%E5%85%B3%E4%BA%8Enpx.html"
  },
  {
    name: "v-6cf5c993",
    path: "/development-process/NPM/%E5%85%B3%E4%BA%8E%20package.json.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6cf5c993").then(next)
    },
  },
  {
    path: "/development-process/NPM/关于 package.json.html",
    redirect: "/development-process/NPM/%E5%85%B3%E4%BA%8E%20package.json.html"
  },
  {
    path: "/development-process/NPM/关于 package.json.html",
    redirect: "/development-process/NPM/%E5%85%B3%E4%BA%8E%20package.json.html"
  },
  {
    name: "v-0c739d55",
    path: "/development-process/mock/%E5%9C%A8%E7%BA%BF%20mock%20server.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0c739d55").then(next)
    },
  },
  {
    path: "/development-process/mock/在线 mock server.html",
    redirect: "/development-process/mock/%E5%9C%A8%E7%BA%BF%20mock%20server.html"
  },
  {
    path: "/development-process/mock/在线 mock server.html",
    redirect: "/development-process/mock/%E5%9C%A8%E7%BA%BF%20mock%20server.html"
  },
  {
    name: "v-63d7fe95",
    path: "/engineering/webpack/bundler%E6%BA%90%E7%A0%81%E7%BC%96%E5%86%99.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-63d7fe95").then(next)
    },
  },
  {
    path: "/engineering/webpack/bundler源码编写.html",
    redirect: "/engineering/webpack/bundler%E6%BA%90%E7%A0%81%E7%BC%96%E5%86%99.html"
  },
  {
    path: "/engineering/webpack/bundler源码编写.html",
    redirect: "/engineering/webpack/bundler%E6%BA%90%E7%A0%81%E7%BC%96%E5%86%99.html"
  },
  {
    name: "v-d3cfdc90",
    path: "/development-process/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95/jest/jest.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d3cfdc90").then(next)
    },
  },
  {
    path: "/development-process/单元测试/jest/jest.html",
    redirect: "/development-process/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95/jest/jest.html"
  },
  {
    path: "/development-process/单元测试/jest/jest.html",
    redirect: "/development-process/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95/jest/jest.html"
  },
  {
    name: "v-4742988c",
    path: "/development-process/%E6%8A%80%E6%9C%AF%E5%9F%BA%E5%BB%BA/%E5%A6%82%E4%BD%95%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AAUI%E7%BB%84%E4%BB%B6%E5%BA%93.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4742988c").then(next)
    },
  },
  {
    path: "/development-process/技术基建/如何搭建一个UI组件库.html",
    redirect: "/development-process/%E6%8A%80%E6%9C%AF%E5%9F%BA%E5%BB%BA/%E5%A6%82%E4%BD%95%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AAUI%E7%BB%84%E4%BB%B6%E5%BA%93.html"
  },
  {
    path: "/development-process/技术基建/如何搭建一个UI组件库.html",
    redirect: "/development-process/%E6%8A%80%E6%9C%AF%E5%9F%BA%E5%BB%BA/%E5%A6%82%E4%BD%95%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AAUI%E7%BB%84%E4%BB%B6%E5%BA%93.html"
  },
  {
    name: "v-2acb4106",
    path: "/development-process/%E6%96%87%E6%A1%A3/storybook.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2acb4106").then(next)
    },
  },
  {
    path: "/development-process/文档/storybook.html",
    redirect: "/development-process/%E6%96%87%E6%A1%A3/storybook.html"
  },
  {
    path: "/development-process/文档/storybook.html",
    redirect: "/development-process/%E6%96%87%E6%A1%A3/storybook.html"
  },
  {
    name: "v-6f5865c1",
    path: "/engineering/webpack/createReactApp%20wepack%E9%85%8D%E7%BD%AE%E5%88%86%E6%9E%90.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6f5865c1").then(next)
    },
  },
  {
    path: "/engineering/webpack/createReactApp wepack配置分析.html",
    redirect: "/engineering/webpack/createReactApp%20wepack%E9%85%8D%E7%BD%AE%E5%88%86%E6%9E%90.html"
  },
  {
    path: "/engineering/webpack/createReactApp wepack配置分析.html",
    redirect: "/engineering/webpack/createReactApp%20wepack%E9%85%8D%E7%BD%AE%E5%88%86%E6%9E%90.html"
  },
  {
    name: "v-5b14736c",
    path: "/engineering/webpack/webpack%E6%80%A7%E8%83%BD%E4%BC%98%E5%8C%96.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5b14736c").then(next)
    },
  },
  {
    path: "/engineering/webpack/webpack性能优化.html",
    redirect: "/engineering/webpack/webpack%E6%80%A7%E8%83%BD%E4%BC%98%E5%8C%96.html"
  },
  {
    path: "/engineering/webpack/webpack性能优化.html",
    redirect: "/engineering/webpack/webpack%E6%80%A7%E8%83%BD%E4%BC%98%E5%8C%96.html"
  },
  {
    name: "v-5e2ae6bb",
    path: "/engineering/webpack/webpack%E5%9F%BA%E7%A1%80%E5%AD%A6%E4%B9%A0.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5e2ae6bb").then(next)
    },
  },
  {
    path: "/engineering/webpack/webpack基础学习.html",
    redirect: "/engineering/webpack/webpack%E5%9F%BA%E7%A1%80%E5%AD%A6%E4%B9%A0.html"
  },
  {
    path: "/engineering/webpack/webpack基础学习.html",
    redirect: "/engineering/webpack/webpack%E5%9F%BA%E7%A1%80%E5%AD%A6%E4%B9%A0.html"
  },
  {
    name: "v-1684c739",
    path: "/engineering/webpack/webpack%E5%AE%9E%E4%BE%8B%E5%BA%94%E7%94%A8.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1684c739").then(next)
    },
  },
  {
    path: "/engineering/webpack/webpack实例应用.html",
    redirect: "/engineering/webpack/webpack%E5%AE%9E%E4%BE%8B%E5%BA%94%E7%94%A8.html"
  },
  {
    path: "/engineering/webpack/webpack实例应用.html",
    redirect: "/engineering/webpack/webpack%E5%AE%9E%E4%BE%8B%E5%BA%94%E7%94%A8.html"
  },
  {
    name: "v-6f25947a",
    path: "/engineering/webpack/webpack%E9%AB%98%E7%BA%A7%E6%A6%82%E5%BF%B5.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6f25947a").then(next)
    },
  },
  {
    path: "/engineering/webpack/webpack高级概念.html",
    redirect: "/engineering/webpack/webpack%E9%AB%98%E7%BA%A7%E6%A6%82%E5%BF%B5.html"
  },
  {
    path: "/engineering/webpack/webpack高级概念.html",
    redirect: "/engineering/webpack/webpack%E9%AB%98%E7%BA%A7%E6%A6%82%E5%BF%B5.html"
  },
  {
    name: "v-324defec",
    path: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-324defec").then(next)
    },
  },
  {
    path: "/framework/React/关于react-hook.html",
    redirect: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html"
  },
  {
    path: "/framework/React/关于react-hook.html",
    redirect: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html"
  },
  {
    name: "v-dfc908d6",
    path: "/engineering/webpack/%E7%BC%96%E4%B8%80%E4%B8%AAwebapck%20loader%E5%92%8Cpugin.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-dfc908d6").then(next)
    },
  },
  {
    path: "/engineering/webpack/编一个webapck loader和pugin.html",
    redirect: "/engineering/webpack/%E7%BC%96%E4%B8%80%E4%B8%AAwebapck%20loader%E5%92%8Cpugin.html"
  },
  {
    path: "/engineering/webpack/编一个webapck loader和pugin.html",
    redirect: "/engineering/webpack/%E7%BC%96%E4%B8%80%E4%B8%AAwebapck%20loader%E5%92%8Cpugin.html"
  },
  {
    name: "v-5941f025",
    path: "/framework/React/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5941f025").then(next)
    },
  },
  {
    path: "/framework/React/index.html",
    redirect: "/framework/React/"
  },
  {
    name: "v-15d6af00",
    path: "/runnnig-monitoring/%E5%AE%89%E5%85%A8/CSRF.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-15d6af00").then(next)
    },
  },
  {
    path: "/runnnig-monitoring/安全/CSRF.html",
    redirect: "/runnnig-monitoring/%E5%AE%89%E5%85%A8/CSRF.html"
  },
  {
    path: "/runnnig-monitoring/安全/CSRF.html",
    redirect: "/runnnig-monitoring/%E5%AE%89%E5%85%A8/CSRF.html"
  },
  {
    name: "v-65b2867d",
    path: "/syntax&API/CSS/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-65b2867d").then(next)
    },
  },
  {
    path: "/syntax&API/CSS/css单位中px和em,rem的区别.html",
    redirect: "/syntax&API/CSS/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html"
  },
  {
    path: "/syntax&API/CSS/css单位中px和em,rem的区别.html",
    redirect: "/syntax&API/CSS/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html"
  },
  {
    name: "v-4050943c",
    path: "/syntax&API/HTML/%E5%85%B3%E4%BA%8EHTML.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4050943c").then(next)
    },
  },
  {
    path: "/syntax&API/HTML/关于HTML.html",
    redirect: "/syntax&API/HTML/%E5%85%B3%E4%BA%8EHTML.html"
  },
  {
    path: "/syntax&API/HTML/关于HTML.html",
    redirect: "/syntax&API/HTML/%E5%85%B3%E4%BA%8EHTML.html"
  },
  {
    name: "v-9176a9d4",
    path: "/tool&plugin/cross-env/%E5%85%B3%E4%BA%8Ecross-env.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-9176a9d4").then(next)
    },
  },
  {
    path: "/tool&plugin/cross-env/关于cross-env.html",
    redirect: "/tool&plugin/cross-env/%E5%85%B3%E4%BA%8Ecross-env.html"
  },
  {
    path: "/tool&plugin/cross-env/关于cross-env.html",
    redirect: "/tool&plugin/cross-env/%E5%85%B3%E4%BA%8Ecross-env.html"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
