// _@vuepress_core@1.7.1@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/licop/Documents/GitHub/What_is_FE/node_modules/_@vuepress_core@1.7.1@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-21ce6dec",
    path: "/runnnig-monitoring/%E6%B5%8F%E8%A7%88%E5%99%A8%E5%92%8C%20webview.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-21ce6dec").then(next)
    },
  },
  {
    path: "/runnnig-monitoring/浏览器和 webview.html",
    redirect: "/runnnig-monitoring/%E6%B5%8F%E8%A7%88%E5%99%A8%E5%92%8C%20webview.html"
  },
  {
    path: "/runnnig-monitoring/浏览器和 webview.html",
    redirect: "/runnnig-monitoring/%E6%B5%8F%E8%A7%88%E5%99%A8%E5%92%8C%20webview.html"
  },
  {
    name: "v-3d79a3a3",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3d79a3a3").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-5941f025",
    path: "/framework/React/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5941f025").then(next)
    },
  },
  {
    path: "/framework/React/index.html",
    redirect: "/framework/React/"
  },
  {
    name: "v-6403c5e2",
    path: "/syntax&API/html/%E5%85%B3%E4%BA%8EHTML.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6403c5e2").then(next)
    },
  },
  {
    path: "/syntax&API/html/关于HTML.html",
    redirect: "/syntax&API/html/%E5%85%B3%E4%BA%8EHTML.html"
  },
  {
    path: "/syntax&API/html/关于HTML.html",
    redirect: "/syntax&API/html/%E5%85%B3%E4%BA%8EHTML.html"
  },
  {
    name: "v-324defec",
    path: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-324defec").then(next)
    },
  },
  {
    path: "/framework/React/关于react-hook.html",
    redirect: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html"
  },
  {
    path: "/framework/React/关于react-hook.html",
    redirect: "/framework/React/%E5%85%B3%E4%BA%8Ereact-hook.html"
  },
  {
    name: "v-f6c8cb06",
    path: "/syntax&API/css/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f6c8cb06").then(next)
    },
  },
  {
    path: "/syntax&API/css/css单位中px和em,rem的区别.html",
    redirect: "/syntax&API/css/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html"
  },
  {
    path: "/syntax&API/css/css单位中px和em,rem的区别.html",
    redirect: "/syntax&API/css/css%E5%8D%95%E4%BD%8D%E4%B8%ADpx%E5%92%8Cem,rem%E7%9A%84%E5%8C%BA%E5%88%AB.html"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
